shared/chat/conversation/header.desktop.js

```diff
@@ -1,13 +1,16 @@
 // @flow
 import React from 'react'
-import {Box, Text} from '../../common-adapters'
-import {globalStyles, globalColors} from '../../styles'
+import {Box, Icon, Usernames} from '../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
 
 import type {Props} from './'
 
 const Header = ({participants}: Props) => (
   <Box style={containerStyle}>
-    <Text type='Body'>{participants.join(', ')}</Text>
+    <Usernames inline={true} type='BodyBig' users={participants.filter(p => !p.you).toJS()}
```

Comment:
🐑  a `.toArray` is probably sufficient and a little faster since it isn't recursive